README.md

```diff
@@ -114,8 +114,9 @@ Then download the source code of the OpenWhisk CLI and the dependencies by
 typing:
 
 ```
-$ cd $GOPATH
-$ go get github.com/apache/incubator-openwhisk-cli
+$ mkdir -p $GOPATH/github.com/apache
+$ cd $GOPATH/github.com/apache
+$ git clone https://github.com/apache/incubator-openwhisk-cli
```

Comment:
i recommend retaining the "go" flavor of this installation, so instead of the `git` command, how about just using the `-d` (download only; no compile/install) option.
```
go get -d github.com/apache/incubator-openwhisk-cli
```
this should avoid the
```
src\github.com\apache\incubator-openwhisk-cli\wski18n\i18n.go:98:16: undefined: Asset
```
error seen without the `-d` option